examples/text_classification/utils/imdb_format.py

```diff
@@ -0,0 +1,112 @@
+# coding=utf-8
+# Copyright 2019 The Google UDA Team Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Read all data in IMDB and merge them to a csv file."""
+
+import csv
+import os
+import argparse
+
+parser = argparse.ArgumentParser(description="Process input and output")
+parser.add_argument('--raw_data_dir', help='raw data directory')
+parser.add_argument('--output_dir', help='output directory')
+parser.add_argument('--train_id_path', help='path of id list')
+args = parser.parse_args()
+
+
+def dump_raw_data(contents, file_path):
+    with open(file_path, "w", encoding="utf-8") as ouf:
+        writer = csv.writer(ouf, delimiter="\t", quotechar="\"")
+        for line in contents:
+            writer.writerow(line)
+
+
+def clean_web_text(st):
```

Comment:
This is still very non-forte. Are we not going to change this?

Reply:
I made a pipeline to save the raw data to CSV. We can use it here.

OR we can assume the user has the CSV format of the dataset, as well as the back-translated text. Since our goal is to demo UDA here, I think we can provide the files for download.

What do you think?

Reply:
Both are very helpful! The pipeline can be used for people who with the raw data. The downloaded data will be useful for users who just want to see the results. Is it possible that we have both?

Reply:
Sure. 

If the user wants to play with the raw data, the user will need to generate back-translation data themselves, though.